README.md

```diff
@@ -0,0 +1 @@
+This is a Microverse project and consist of producing a clone of Mint.com’s signup page using html and css. To view this project life, you can visit the link https://roynyaga.github.io/microverse_html_form_project/
```

Comment:
`SUGGESTION`

Nice job writing a descriptive `README.md` file for describing what the project does. You can even add other information like live page link, assignment link, collaborators and a screenshot of the webpage. :+1: :star2: